src/store.ts

```diff
@@ -11,35 +11,62 @@ interface ICycle {
 }
 
 const startingCycles: ICycle[] = [
-  {cursor: 2, length: 2},
+  { cursor: 1, length: 2 },
+  { cursor: 3, length: 3 },
 ];
 
 export default new Vuex.Store({
   state: {
     asm: {
       activators: 1,
-      // cycles: new Array<ICycle>(),
+      nonActivators: 1,
       cycles: startingCycles,
       iteration: 0,
     },
   },
   getters: {
     anyActive(state): boolean {
-      return _.some(state.asm.cycles, (cycle) => {
-        return cycle.cursor <= state.asm.activators;
+      var isActive = false;
+      state.asm.cycles.forEach((cycle) => {
+        if (cycle.cursor <= state.asm.activators) {
+          isActive = true;
+        }
       });
+      return isActive;
     },
   },
   mutations: {
     setActivators(state, payload: number) {
-      state.asm.activators = payload;
+      state.asm.activators = Number(payload);
+      const cycleLength = state.asm.activators + state.asm.nonActivators;
+      state.asm.cycles = [
+        { cursor: 1, length: cycleLength },
+        { cursor: cycleLength + 1, length: cycleLength + 1 },
+      ];
+      state.asm.iteration = 0;
+    },
+    setNonActivators(state, payload: number) {
```

Comment:
@navyhockey56 I think this mutation shouldn't touch `asm.cycles` or `asm.iteration`, these belong in their own mutations.

Reply:
@sballan I wasn't sure what the proper way of doing that would be, I'll look more into mutations and get that working as you've suggested

Reply:
@navyhockey56 What do you think of making a `dev` branch, where we sketch out new ideas/features? Might be faster to iterate that way, and then do a thorough review of the `dev` branch before merging it to master. 

I marked up this PR as if it were going on master, but I feel like issues like this ^ can be part of a larger conversation about the design of this app.  Like - do we really need to get `mutators` vs `actions` etc perfectly just to get out a few features on a `dev` branch we can see working somewhere?

Just thinking out loud.

Reply:
@sballan Would the workflow be:
A. branch gets promoted to DEV, DEV gets promoted to master
B. branch gets promoted to DEV, branch gets promoted to master
C. Something else?

Reply:
@sballan Also, I was playing around with trying to move that functionality inside of the cycle itself, but hit a "I don't know nearly enough about vue" wall :/

Reply:
@navyhockey56 I was thinking A, and yea I was wondering about logic in the store vs outside of it...I figured 'everything in the store' first, but I think it would be cool to demo a few different implementations.